app/src/main/java/dev/arkbuilders/rate/presentation/utils/NotificationUtils.kt

```diff
@@ -35,10 +39,18 @@ object NotificationUtils {
         createNotificationChannel(ctx)
 
         with(NotificationManagerCompat.from(ctx)) {
-            notify(pairAlertCondition.id.toInt(), builder.build())
+
+            if (ActivityCompat.checkSelfPermission(ctx,
+                                                   Manifest.permission.POST_NOTIFICATIONS) == PackageManager.PERMISSION_GRANTED
+            ) {
+
+                notify(pairAlertCondition.id.toInt(), builder.build())
+            }
+
```

Comment:
I understand that this snippet is for preventing runtime crash related to permission in Android 13. 
But there are 2 problems:

1/ It's not mentioned in the PR's description.
2/ This check should only be there for API 31 or up.

Reply:
it's pointless to make a check for a check because the permission is always granted for API 32 and lower.

Reply:
Related? https://github.com/ARK-Builders/ARK-Rate/issues/45